### ISSUE TITLE
PluginNotFoundException

### ISSUE THREAD

**Issue by pgerbrands** [keitaroinc/ckanext-s3filestore#10]
**Describe the bug**
After the installation (following the exact installation procedure as shown) restarting the CKAN server results in the error: ckan.plugins.core.PluginNotFoundException: s3filestore. I would like to integrate the MinIO storage that I have confgured (I have set all the variables as defined after the installation procedure, but I have not been able to test it yet due to this error message). The ckan-worker-00 process is therefore not started by supervisor.

**ckanext-s3filestore version affected**
v0.0.5

**Expected behaviour**
Normal start of the environment.

**Logs**
2022-02-22 15:24:44,821 INFO  [ckan.cli] Using configuration file /etc/ckan/default/ckan.ini
2022-02-22 15:24:44,821 INFO  [ckan.config.environment] Loading static files from public
2022-02-22 15:24:44,850 INFO  [ckan.config.environment] Loading templates from /usr/lib/ckan/default/src/ckan/ckan/templates
Traceback (most recent call last):
  File "/usr/lib/ckan/default/bin/ckan", line 11, in <module>
    load_entry_point('ckan', 'console_scripts', 'ckan')()
  File "/usr/lib/ckan/default/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/ckan/default/lib/python3.8/site-packages/click/core.py", line 781, in main
    with self.make_context(prog_name, args, **extra) as ctx:
  File "/usr/lib/ckan/default/lib/python3.8/site-packages/click/core.py", line 700, in make_context
    self.parse_args(ctx, args)
  File "/usr/lib/ckan/default/src/ckan/ckan/cli/cli.py", line 115, in parse_args
    result = super(ExtendableGroup, self).parse_args(ctx, args)
  File "/usr/lib/ckan/default/lib/python3.8/site-packages/click/core.py", line 1212, in parse_args
    rest = Command.parse_args(self, ctx, args)
  File "/usr/lib/ckan/default/lib/python3.8/site-packages/click/core.py", line 1048, in parse_args
    value, args = param.handle_parse_result(ctx, opts, args)
  File "/usr/lib/ckan/default/lib/python3.8/site-packages/click/core.py", line 1630, in handle_parse_result
    value = invoke_param_callback(self.callback, ctx, self, value)
  File "/usr/lib/ckan/default/lib/python3.8/site-packages/click/core.py", line 123, in invoke_param_callback
    return callback(ctx, param, value)
  File "/usr/lib/ckan/default/src/ckan/ckan/cli/cli.py", line 125, in _init_ckan_config
    _add_ctx_object(ctx, value)
  File "/usr/lib/ckan/default/src/ckan/ckan/cli/cli.py", line 134, in _add_ctx_object
    ctx.obj = CtxObject(path)
  File "/usr/lib/ckan/default/src/ckan/ckan/cli/cli.py", line 56, in __init__
    self.app = make_app(self.config)
  File "/usr/lib/ckan/default/src/ckan/ckan/config/middleware/__init__.py", line 56, in make_app
    load_environment(conf)
  File "/usr/lib/ckan/default/src/ckan/ckan/config/environment.py", line 123, in load_environment
    p.load_all()
  File "/usr/lib/ckan/default/src/ckan/ckan/plugins/core.py", line 165, in load_all
    load(*plugins)
  File "/usr/lib/ckan/default/src/ckan/ckan/plugins/core.py", line 179, in load
    service = _get_service(plugin)
  File "/usr/lib/ckan/default/src/ckan/ckan/plugins/core.py", line 282, in _get_service
    raise PluginNotFoundException(plugin_name)
ckan.plugins.core.PluginNotFoundException: s3filestore

**Comment by jeverling** [comment]
Hi, maybe the plugin didn't get installed into the Python environment correctly? What is the output of `pip freeze | grep s3filestore`?

**Comment by pgerbrands** [comment]
Hi Jesaja, thanks for your quick answer.- The output is:

-e git+https://github.com/okfn/ckanext-s3filestore.gt@<<some hash key?>>#egg=ckanext_s3filestore

**Comment by pgerbrands** [comment]
Dear Jesaja,

Most likely I have solve the situation: I guess three things happened: 
1) the user did not have write access to /usr/lib/ckan/  --> so I chowned the directory (recursively).
2) I used "sudo" before the pip install command (therefore I did not see the previous error).
3) I received an error with the pip command "error: invalid command "bdist_wheel' --> thus "pip install wheel" solved it

Fixing these three things made the PluginNotFoundException error disappear.
(Now I only have an EndpointConnectionError to my MinIO server, but I hope to figure this out soon).

So my apologies for bothering you, the error was my own mistake...
